GOVERNANCE.md

```diff
@@ -50,64 +69,76 @@ For nominations, the maintainers will look at the following criteria:
+Any maintainer may suggest a request for CNCF resources, in the
 [developer mailing list](https://groups.google.com/forum/#!forum/kubevirt-dev), 
-the [Maintainer mailing list](mailto:cncf-kubevirt-maintainers@lists.cncf.io), on Github, 
-or during a community meeting.  A simple majority of Maintainers approves the 
-request.  The Maintainers may also choose to delegate working with the CNCF to 
-non-Maintainer community members.
+the [maintainer mailing list](mailto:cncf-kubevirt-maintainers@lists.cncf.io), on Github, 
+or during a community meeting.  A simple majority of maintainers approves the 
+request.  The maintainers may also choose to delegate working with the CNCF to 
+non-maintainer community members.
 
 ### Code of Conduct
 
 [Code of Conduct](./code-of-conduct.md)
 violations by community members will be discussed and resolved
-on the [private Maintainer mailing list](mailto:cncf-kubevirt-maintainers@lists.cncf.io).  If the reported CoC violator
-is a Maintainer, the Maintainers will instead designate two Maintainers to work
+on the [private maintainer mailing list](mailto:cncf-kubevirt-maintainers@lists.cncf.io).  If the reported CoC violator
+is a maintainer, the maintainers will instead designate two maintainers to work
 with CNCF staff in resolving the report.
 
-### Removing Maintainers
+### Off-boarding Maintainers and Mentorship
+
+Maintainers can voluntarily retire at any time, and should always feel that they can rely
+on others to continue their good work. When a maintainer leaves, the project loses valuable 
+knowledge and history.
+
+To facilitate this process, it is encouraged that maintainers recognise when they no longer
+have sufficient time to give to the project and engage in a mentorship with a possible
+maintainer candidate from the community. Through this process, they can help transition some of their experience
+and guide the candidate through expectations of the role.
+
+### Retiring and Removing Maintainers
 
-Maintainers may voluntarily retire at any time.  Should a maintainer retire, 
-it requires a majority vote of the current maintainers to reinstate them.
+A maintainer can retire or be removed as a maintainer by filing an PR in the
+[Community repo](https://github.com/kubevirt/community) against the [MAINTAINERS](./MAINTAINERS.md) file and moving to Emeritus.
 
 Maintainers may also be demoted at any time for one of the following reasons:
 
 * Inactivity, including 6 months or more of non-participation or non-communication,
 * Refusal to abide by this Governance,
 * Violations of the Code of Conduct,
-* Other actions that harm the reputation, stability, or harmony of the Kubevirt
+* Other actions that harm the reputation, stability, or harmony of the KubeVirt
   project.
 
 Removing a maintainer requires a 2/3 majority vote of the other maintainers.
+Should a maintainer retire or be demoted and wish to return, it requires a majority vote of the current maintainers to reinstate them.
```

Comment:
Perhaps we could add something like: "Although requiring a vote by maintainer, it should be much easier for an emeritus maintainer to return becoming a maintainer"?